Changelog.md

```diff
@@ -26,6 +26,10 @@ changed. diaspora\* will no longer listen on `0.0.0.0:3000` as it will now
 bind to an UNIX socket at `unix:tmp/diaspora.sock`. Please change your local
 `diaspora.yml` if necessary.
 
+## Terms of Use design changes
+
+With the port to Bootstrap 3, the app/views/terms/default.haml file changed. If you edited the terms by adding a terms.erb file, you probably want to backup these changes in it
```

Comment:
In https://github.com/diaspora/diaspora/blob/develop/config/diaspora.yml.example#L496 we mention `app/views/terms/terms.haml` and `app/views/terms/terms.erb` so we should also mention both here.

I'm also not sure about the wording "backup these changes in it". `app/views/terms/default.haml` now has a new structure and the podmin should make sure that his/hers terms file has the same structure. Maybe @goobertron could help us here.